index.js

```diff
@@ -119,25 +119,14 @@ function parseOpts (opts) {
   // Add user ignore patterns to default ignore patterns
   var ignore = (opts.ignore || []).concat(DEFAULT_IGNORE_PATTERNS)
 
-  var root
-  try {
-    root = findRoot(opts.cwd)
```

Comment:
This is where we used to pass in `opts.cwd`. But that's no longer used with your changes. Please fix.